packages/api-sdk/.speakeasy/gen.yaml

```diff
@@ -20,6 +20,9 @@ generation:
     oAuth2ClientCredentialsEnabled: false
     oAuth2PasswordEnabled: false
 typescript:
+  compileCommand:
```

Comment:
the Speakeasy team said this should allow us to use `pnpm` instead of `npm`. i'm going to handle updating Speakeasy sometime this week. reference docs: [here](https://www.speakeasy.com/guides/sdks/override-compile#overriding-compile-commands-for-sdk-generation).

we previously saw problems when i deleted the `package-lock.json` file, so i'll keep an eye out and can revert this, if needed. i gave it a test locally, and it seemed to work fine.